privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/workers/RemoteConfigDownloadWorker.kt

```diff
@@ -54,19 +56,24 @@ class PrivacyConfigDownloadWorker(
     }
 }
 
-@ContributesMultibinding(AppScope::class)
+@ContributesMultibinding(
+    scope = AppScope::class,
+    boundType = LifecycleObserver::class
+)
+@SingleInstanceIn(AppScope::class)
 class PrivacyConfigDownloadWorkerScheduler @Inject constructor(
     private val workManager: WorkManager
-) : LifecycleObserver {
+) : LifecycleEventObserver {
```

Comment:
nit: if you extend `DefaultLifecycleObserver` you can then override `onCreate` function instead, rather than the whole `if (event == ...)` thing

Reply:
Done!